docs/snippets/ov_infer_request.py

```diff
@@ -3,12 +3,12 @@
 
 import numpy as np
 #! [import]
-import openvino.runtime as ov
+import openvino as ov
 #! [import]
 
 core = ov.Core()
 model = core.read_model("model.xml")
-compiled_model = core.compile_model(model, "AUTO")
+compiled_model = ov.compile_model(model, "AUTO")
```

Comment:
For example, here you either go with `Core` path or `ov.compile_model` as the second one is actually reading model.